imageGenSD.py

```diff
@@ -104,16 +122,17 @@ async def setup_hook(self):
 
             Output : None
         """
-        
+
         #Replies should be managed in a separate task to allow the main UI to
         #always be responsive, and allow the backend to process work independent
         #of message posting.  It's more efficent and better separated.
         self.loop = asy.get_running_loop()
-        
+
         self.disLog.debug(f"Syncing Guild Tree to Global.")
-        
-        await self.tree.sync()
-        
+        #self.tree.copy_global_to(guild=dis.Object(id=1084545432253894727))
```

Comment:
Note for commented out code, add description if needed

Reply:
Nah, removed.  It's how you can force a slash command sync instead of waiting up to an hour for Discord to sync slash commands globally.  It's also how the server somehow ends up with two copies of the command, and it wasn't very clear [from the example](https://github.com/Rapptz/discord.py/blob/v2.3.2/examples/app_commands/basic.py) how to fix it.

Fortunately it only janks up the dev server so it's a w/e.